services/core/src/sourcify-chains.ts

```diff
@@ -148,5 +148,22 @@ export default {
         "supported": true,
         "monitored": true,
         "graphQLFetchAddress": "https://rinkeby-indexer.arbitrum.io/graphql"
+    "40": {
+        "fullnode": {
+            "dappnode": "https://mainnet.telos.net/evm"
+        },
+        "supported": true,
+        "monitored": false,
+        "contractFetchAddress": "https://bscscan.com/" + ETHERSCAN_SUFFIX,
```

Comment:
[My previous comment](https://github.com/ethereum/sourcify/pull/487#issuecomment-868322676) about explorers concerns this line. In case a contract uses immutable state variables, Sourcify needs to know the chain's block explorer to be able to correctly verify. So I guess this will have to be changed to something other than the Binance Smart Chain explorer.

Reply:
Fixed.